NineChronicles.Headless.Executable/Commands/ChainCommand.cs

```diff
@@ -71,7 +71,7 @@ public void Help([FromService] ICoconaHelpMessageBuilder helpMessageBuilder)
                                "mimisbrunnr) of a given chain in csv format.")]
         public void Inspect(
             [Argument("STORE-TYPE",
-                Description = "Store type of RocksDb.")]
+                Description = "Store type of blockchain storage.")]
```

Comment:
I don't have a detailed plan but I think it'll be better to restrict the `Memory` store type option here yet.

Reply:
So, you mean it'll be better to leave this description to unchanged?
I changed this because of the default option(`LiteDB`) can be used.
How about adding available options to description?

Reply:
Oops, sorry for confusing you with the selected lines. I mean that `MemoryStore` will be useless in the command because the `MemoryStore` is volatile, so I suggested restricting the type 🤔